follow/models.py

```diff
@@ -168,3 +168,13 @@ def cancel(self):
         request_cancel.send(sender=self)
         self.delete()
         return True
+
+
+class RemoteFollow(models.Model):
```

Comment:
```suggestion
class RemoteFollower(models.Model):
```

Reply:
Lol, I'm so overwhelming right now. Thanks for pointing out!

Reply:
No worries no worries, thanks for pushing this through hahahah